docker/beta/api_compat/consul-template-api-compat.conf

```diff
@@ -0,0 +1,12 @@
+template {
+    source = "/code/listenbrainz/consul_config.py.ctmpl"
+    destination = "/code/listenbrainz/listenbrainz/config.py"
+}
+
+exec {
+    command = "uwsgi /etc/uwsgi/uwsgi-api-compat.ini"
+    splay = "60s"
+    reload_signal = "SIGHUP"
```

Comment:
we need to double-check all of these signals to determine that the child process does the right thing. I just copied these from the musicbrainz config file

Reply:
According to https://uwsgi-docs.readthedocs.io/en/latest/Management.html we either use `SIGINT` to kill uwsgi, or run it as `uwsgi --die-on-term` to have it accept `SIGTERM` as the shutdown command